controllers/flinkcluster_util.go

```diff
@@ -484,6 +490,7 @@ func isUpdatedAll(observed ObservedClusterState) bool {
 	return areComponentsUpdated(components, *observed.cluster)
 }
 
+// isClusterUpdated checks whether all cluster components are replaced to next revision.
 func isClusterUpdated(observed ObservedClusterState) bool {
```

Comment:
Seems for a cluster which has never been updated, this function always returns true, but the function name feels like "an update has been triggered and finished". Can we rename it to `isClusterUpdateToDate`?

Reply:
Done.